.github/workflows/release.yml

```diff
@@ -1,38 +1,37 @@
-# .github/workflows/release.yml
-name: goreleaser
+name: Release
 
 on:
   push:
     # run only against tags
     tags:
-      - "*"
+      - "v*"
```

Comment:
This project doesn't have any tags that start with "v". This will never run.

Reply:
will remove